alf/utils/resnet50.py

```diff
@@ -0,0 +1,91 @@
+# Copyright (c) 2019 Horizon Robotics. All Rights Reserved.
```

Comment:
The filename would be better called "resnet50_block.py"

Reply:
renamed